<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDlgReal</class>
 <widget class="QDialog" name="RegisterDlgReal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/img/bg.jpg);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Title">
      <property name="styleSheet">
       <string notr="true">#Title{
	border-image: url(:/img/img/neutalk.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#register_button{
	border-image: url(:/img/img/color.png);
border-radius:10px;
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passward_enter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/black.jpg);
color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>   Enter a password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passward_confirm">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/black.jpg);
color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>   Confirm the password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username_enter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/black.jpg);
color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>   Enter your username</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
